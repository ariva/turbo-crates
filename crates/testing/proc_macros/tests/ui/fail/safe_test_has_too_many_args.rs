use turbo_crates_testing_proc_macros::*;

fn main() {}

#[safe_tests]
pub(crate) mod test {
    const TEST_ENV_VAR: &str = "TEST_ENV_VAR";

    #[safe_test(a, b)]
    fn it_works1() {
        std::env::set_var(TEST_ENV_VAR, "test1");
        let test_value = std::env::var(TEST_ENV_VAR).unwrap();
        assert_eq!(test_value, "test1");
    }
}
