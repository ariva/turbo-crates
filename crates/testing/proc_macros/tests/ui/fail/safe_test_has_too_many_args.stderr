error: custom attribute panicked
 --> tests/ui/fail/safe_test_has_too_many_args.rs:9:5
  |
9 |     #[safe_test(a, b)]
  |     ^^^^^^^^^^^^^^^^^^
  |
  = help: message: `"a, b"` is not a valid identifier
